src/test/java/com/agilethought/atsceapi/service/UserServiceImplTest.java

```diff
@@ -1,12 +1,16 @@
 package com.agilethought.atsceapi.service;
 
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
```

Comment:
it is not recommended include all packages inside of library. just import packages that are used